transform/snowflake-dbt/models/blapi/customers_with_cloud_paid_subs.sql

```diff
@@ -35,7 +35,7 @@ with current_subscriptions AS(
         current_subscriptions.company,
         left(coalesce(current_subscriptions.company, customers.email), 40) as company_name,
         current_subscriptions.status,
-        to_varchar(current_subscriptions.start_date, 'yyyy-MM-dd HH:mm:ss.SSS Z') as start_date,
+        to_varchar(current_subscriptions.DATE_CONVERTED_TO_PAID::int) as start_date,
```

Comment:
Shouldn't date be a date, rather than a varchar?

Reply:
its how its being used previously to sync to salesforce, so.